### PR TITLE
Fix link to testnet faucets

### PR DESCRIPTION
**Pull request by jpuri**
## **Description**

Fix the link to testnet faucets.

## **Manual testing steps**

_1. Start a transaction and go to confirmation page (ensure that you have less that required ETH on the testnet)
_2. Click the **buy** option, it should load correct faucet page

## **Related issues**

_Fixes: MetaMask/mobile-planning#1255

## **Pre-merge author checklist**

- [X] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [X] I've clearly explained:
  - [X] What problem this PR is solving.
  - [X] How this problem was solved.
  - [X] How reviewers can test my changes.
- [X] I’ve indicated what issue this PR is linked to: Fixes #???
- [ ] I’ve included tests if applicable.
- [ ] I’ve documented any added code.
- [X] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
- [X] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [X] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by segun — DISMISSED** [review]

File: app/core/AppConstants.ts [hunk 1]
```diff
@@ -88,7 +88,8 @@ export default {
       'https://metamask.zendesk.com/hc/en-us/articles/360015489591-Basic-Safety-and-Security-Tips-for-MetaMask',
     TOKEN_BALANCE:
       'https://metamask.zendesk.com/hc/en-us/articles/360028059272-What-to-do-when-your-balance-of-ETH-and-or-ERC20-tokens-is-incorrect-inaccurate',
-    MM_FAUCET: 'https://faucet.metamask.io/',
+    SEPOLIA_FAUCET: 'https://www.infura.io/faucet/sepolia',
+    LINEA_FAUCET: 'https://www.infura.io/faucet/linea',
```
Comment (blackdevelopa) [inline app/core/AppConstants.ts:92]: Should we include `GOERLI_FAUCET` url?

Comment (blackdevelopa) [reply app/core/AppConstants.ts:92]: Can we move these network URLs to `../constants/url` under `// Network` section

**Review by blackdevelopa — DISMISSED** [review]
Looks good to me.

**Review by digiwand — DISMISSED** [review]

**Review by blackdevelopa — APPROVED** [review]

**Review by segun — APPROVED** [review]

**Comment by jpuri** [comment]
Bitrise: https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/269de88a-95d8-4835-89e5-73a75aa60148

**Pull request merged**